packages/components/Tabs/src/Tabs.types.ts

```diff
@@ -38,11 +38,20 @@ export interface TabsContextData {
    * A Map to store the corresponding view of each TabsItem
    */
   views?: Map<string, React.ReactNode[]> | null;
+
+  /**
+   * GH #964, Reference to the Focus Container as there is no FocusZone on windows.
+   */
+  focusZoneRef?: React.RefObject<any> | null;
```

Comment:
I would add another line to the comments, something like: "Windows-Specific Prop`